<ui version="4.0" >
 <class>NewImageDlg</class>
 <widget class="QDialog" name="NewImageDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Image Chooser - New Image</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="5" column="1" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>269</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <widget class="QLabel" name="colorLabel" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QPushButton" name="colorButton" >
     <property name="text" >
      <string>&amp;Color...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QComboBox" name="brushComboBox" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>&amp;Brush pattern:</string>
     </property>
     <property name="buddy" >
      <cstring>brushComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&amp;Width:</string>
     </property>
     <property name="buddy" >
      <cstring>widthSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&amp;Height:</string>
     </property>
     <property name="buddy" >
      <cstring>heightSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QSpinBox" name="heightSpinBox" >
     <property name="alignment" >
      <set>Qt::AlignRight</set>
     </property>
     <property name="suffix" >
      <string> px</string>
     </property>
     <property name="maximum" >
      <number>512</number>
     </property>
     <property name="minimum" >
      <number>8</number>
     </property>
     <property name="singleStep" >
      <number>4</number>
     </property>
     <property name="value" >
      <number>64</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSpinBox" name="widthSpinBox" >
     <property name="alignment" >
      <set>Qt::AlignRight</set>
     </property>
     <property name="suffix" >
      <string> px</string>
     </property>
     <property name="maximum" >
      <number>512</number>
     </property>
     <property name="minimum" >
      <number>8</number>
     </property>
     <property name="singleStep" >
      <number>4</number>
     </property>
     <property name="value" >
      <number>64</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>widthSpinBox</tabstop>
  <tabstop>heightSpinBox</tabstop>
  <tabstop>brushComboBox</tabstop>
  <tabstop>colorButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewImageDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>181</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel" >
     <x>198</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewImageDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>150</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel" >
     <x>159</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
